uber/templates/emails/reg_workflow/attendee_confirmation.html

```diff
@@ -14,6 +22,11 @@
 {% endif %}
 extra money <a href="{{ c.URL_BASE }}/preregistration/confirm?id={{ attendee.id }}">here</a>.
 
+{% if attendee.paid == c.NEED_NOT_PAY and not attendee.promo_code_id %}
```

Comment:
This covers the relatively rare case mentioned in https://github.com/magfest/ubersystem/issues/1709, where an admin fills out all the info for an attendee rather than making them a placeholder badge.